src/table/table-header-item.class.ts

```diff
@@ -69,6 +84,15 @@ export class TableHeaderItem {
 	 */
 	data: any;
 
+	/**
+	 * Data for the header item for general usage in the controller.
+	 * For example, which `field` is this column related to.
+	 *
+	 * @type {*}
+	 * @memberof TableHeaderItem
+	 */
+	additionalData: any;
```

Comment:
This is for non-presentational data that would be used by the controller later on, for example `dbField: "name"` to show which database field that column should be sorting by. The `data` property cannot hold that `dbField` because that would require for me to supply a template for that cell which I would really like not to do as it takes time and it's not relevant to the view at all.

Reply:
It makes sense to have this in those cases. Name it `metadata`, it's more descriptive what it should be used for.